client/boot/app.js

```diff
@@ -1,10 +1,7 @@
 /** @format */
 // Initialize polyfills before any dependencies are loaded
 import './polyfills';
-
-if ( process.env.NODE_ENV === 'development' ) {
-	require( 'lib/wrap-es6-functions' )();
-}
+import 'lib/wrap-es6-functions';
```

Comment:
It would be nice to remove it completely, which should be possible since we started using babel preset env.

Reply:
We aren't using `babel-preset-env` to polyfill yet, though. We're waiting for `babel-preset-env@2`, which can include only the polyfills that actually used & required for the codebase.

Reply:
So it’s fine as it is 👍